tools/coverage/main.go

```diff
@@ -30,27 +30,24 @@ import (
 )
 
 const (
-	keyCovProfileFileName    = "key-cov-prof.txt"
-	defaultStdoutRedirect    = "stdout.txt"
-	defaultCoverageTargetDir = "./pkg/"
-	defaultGcsBucket         = "knative-prow"
-	defaultPostSubmitJobName = ""
-	defaultCovThreshold      = 80
-)
-
-var (
-	artifactsDir        = "./artifacts/"
-	coverageProfileName = "coverage_profile.txt"
+	keyCovProfileFileName      = "key-cov-prof.txt"
+	defaultStdoutRedirect      = "stdout.txt"
+	defaultCoverageTargetDir   = "."
+	defaultGcsBucket           = "knative-prow"
+	defaultPostSubmitJobName   = ""
+	defaultCovThreshold        = 80
```

Comment:
Since you're here, can we make this 50, and then remove all the default values in the Prow config?

Reply:
Just want to make sure if there was any specific reason to set threshold for some repos as `80` instead of `50`? If not, I'm definitely okay with making this change.

Reply:
The original idea was to start with low thresholds, and increase them over time. Anyway, this is a default value, it serves no other purpose than avoiding to override it when the most common case applies. This is definitely this case.

Reply:
Thanks for the explanation. Done.